proposed/2020/Transitive-Dependencies.md

```diff
@@ -133,7 +164,7 @@ The above elements will be checked against Accessibility Insights. We'll ensure
 - A: The NuGet client team should own this work. They are familiar with the Visual Studio Package Manager UI & concepts of transitive dependencies.
 
 - Q: Should packages.config be considered in this work?
-- A: Since packages.config is based on top-level packages only, it will just use its existing experience by placing all packages in the top-level packages section.
+- A: Since packages.config is based on top-level packages only, it will just use its existing experience by placing all packages in the top-level packages section unless the it's technically convenient.
```

Comment:
```suggestion
- A: Since packages.config is based on top-level packages only, it will just use its existing experience by placing all packages in the top-level packages section unless it's technically convenient.
```

Reply:
Removed extra word "the" from the sentence.